app/src/renderer/js/main.js

```diff
@@ -100,6 +100,12 @@ document.addEventListener('DOMContentLoaded', () => {
     cropperBounds.width -= 2;
     cropperBounds.height -= 2;
 
+    // If we're recording fullscreen, set x, y to zero
+    if (dimensions.app && dimensions.app.isFullscreen) {
+      cropperBounds.x = 0;
+      cropperBounds.y = 0;
```

Comment:
The cropper buffer is on all sides, not just the bottom-left. So you need to adjust width and height too. Add 2 px.

Reply:
@sindresorhus that's already done. I tested it the width and height are correct

Reply:
^ the above screen capture shows it